dev-libs/libdivsufsort/libdivsufsort-2.0.1.ebuild

```diff
@@ -1,20 +1,27 @@
-# Copyright 1999-2012 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=4
+EAPI=6
 inherit cmake-utils multilib
 
+if [[ ${PV} != 9999* ]]; then
```

Comment:
I know that it works because of the order. But it looks a little bit like vapier ant not like mgorny ;). I will drop the switch as you wish then.